validators/filterResidentialBuildings/map.js

```diff
@@ -1,15 +1,29 @@
 'use strict';
 var turf = require('@turf/turf');
+var moment = require('moment');
```

Comment:
Have you checked 'date-fns'? I found it to be smaller and have better tools. moment works great too, just letting you know about an alternative.